auto3dseg/algorithm_templates/swinunetr/scripts/train.py

```diff
@@ -162,6 +164,21 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
     model = parser.get_parsed_content("network")
     model = model.to(device)
 
+    #Load pre-trained weights
+    if pretrain_swin["use_pretrain"]:
+        download_url(
```

Comment:
Please set `progress` to `False` for `progress` because tqdm looks bad in the subprocess call. Thanks!

Reply:
No problem, set to False